service-front/module/Application/src/Contracts/OpgApiServiceInterface.php

```diff
@@ -128,4 +128,6 @@ public function updateCaseProgress(string $uuid, array $data): void;
     public function createYotiSession(string $uuid): array;
 
     public function estimatePostofficeDeadline(string $uuid): string;
+
+    public function getServiceAvailability(): array;
```

Comment:
Would be good to include an array shape here so it's clear what we're working with:
```suggestion
    /**
     * @param array<string, bool>
     */
    public function getServiceAvailability(): array;
```